include/meta/meta_libcxx_workaround.hpp

```diff
@@ -0,0 +1,159 @@
+/// \file meta_libcxx_workaround.hpp Make meta::quote work consistently with
+/// libc++ containers
+///
+/// Works around: http://llvm.org/bugs/show_bug.cgi?id=22601
+///  and http://llvm.org/bugs/show_bug.cgi?id=22605 Meta library
+//
+//  Copyright Eric Niebler 2014-2015
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/ericniebler/meta
+//
+#include <meta/meta_fwd.hpp>
```

Comment:
Oops, includes go *after* include guards. Also, meta.hpp should include the workaround. Users shouldn't need to know about it.